stubs/setuptools/setuptools/msvc.pyi

```diff
@@ -17,7 +19,11 @@ class PlatformInfo:
     def cross_dir(self, forcex86: bool = False): ...
 
 class RegistryInfo:
-    HKEYS: Incomplete
+    if sys.platform == "win32":
+        HKEYS: ClassVar[tuple[int, int, int, int]]
```

Comment:
Any reason not to use `Final` here?

Reply:
I think this can be made Final. I doubt it's meant to be overridden by subclasses